bindings_ffi/src/mls.rs

```diff
@@ -1551,6 +1552,7 @@ impl FfiConversations {
                 Err(e) => callback.on_error(e.into()),
             },
             move || close_cb.on_close(),
+            true,
```

Comment:
This should be false. We don't want to stream duplicate dms at the conversation level. Only the messages from those duplicate dms in streamAll.